core/src/blocks/database_schema.rs

```diff
@@ -78,8 +78,19 @@ impl Block for DatabaseSchema {
         ))
         .await?;
 
+        let results = std::iter::zip(schemas, databases)
```

Comment:
@spolu I think we need this, because otherwise it's going to be a massive pain to call the DB block. WDYT